File: Solutions/9109/java/program.java
// Copy from Quera

import java.util.Scanner;

public class Main {
    final static int maxn = 100;
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();
        int[] counter = new int[maxn];
        for (int i = 0; i < n; i++) {
            int x = scanner.nextInt();
            counter[x - 1]++;
        }

        int mn = -1;
        for (int i = 0; i < maxn; i++)
            if (counter[i] > 0 && (mn == -1 || counter[i] < counter[mn]))
                mn = i;

        System.out.println(mn + 1);
    }
}

File: Solutions/9109/c++/program.cpp
// Copy from Quera

#include <iostream>

using namespace std;

const int maxn = 100;
int counter[maxn];

int main()
{
    int n;
    cin >> n;
    for (int i = 0; i < n; i++)
    {
        int x;
        cin >> x;
        counter[x - 1]++;
    }

    int mn = -1;
    for (int i = 0; i < maxn; i++)
        if (counter[i] > 0 && (mn == -1 || counter[mn] > counter[i]))
            mn = i;

    cout << mn + 1 << '\n';
}
